golem-worker-service-base/Cargo.toml

```diff
@@ -14,6 +14,10 @@ harness = false
 name = "services_tests"
 harness = false
 
+[[test]]
+name = "api_gateway_end_to_end_tests"
+harness = false
```

Comment:
New gateway end to end tests. Apparently this PR became more of setting up the stage of further work, but it is really making things easy with these additions